Jenkinsfile

```diff
@@ -61,6 +65,21 @@ pipeline {
                 }
             }
         }
+        stage('Quality Analysis - SonarCloud') {
+            when {
+                allOf {
+                    expression { env.CHANGE_ID == null }
```

Comment:
❓ is this variable set anywhere or is it defaulted by checkout scm?

Reply:
It's not set, so I believe it's checkout scm.